core/templates/pages/contributor-dashboard-page/login-required-message/login-required-message.component.html

```diff
@@ -1,12 +1,8 @@
 <div class="oppia-login-required-message-container">
   <div class="oppia-mascot-avatar-container">
-    <a [ngClass]="{'oppia-disabled-image-link': OPPIA_AVATAR_LINK_URL === null}"
-       [href]="OPPIA_AVATAR_LINK_URL"
-       rel="noopener"
-       target="_blank">
-      <img class="oppia-mascot-avatar"
-           [src]="OPPIA_AVATAR_IMAGE_URL" alt="">
-    </a>
+    <img class="oppia-mascot-avatar"
```

Comment:
Try to go to the contributor dashboard without logging and put a a proof of what you see there.

Reply:
Not addressed.

Reply:
@chris7716 without logging I am unable to go to the contributor dashboard. can you tell me how it is done.

Reply:
does without logging here mean without doing any changes in the file

Reply:
I mean without using the log in to the Oppia application(using email) go to the contributor dashboard and put a screen shot.